tests/saitests/sai_qos_tests.py

```diff
@@ -235,28 +252,34 @@ def runTest(self):
                 send_packet(self, src_port_id, pkt, 1)
                 print >> sys.stderr, "dscp: %d, calling send_packet()" % (tos >> 2)
 
-                cnt = 0
-                dscp_received = False
-                while not dscp_received:
-                    result = self.dataplane.poll(device_number=0, port_number=dst_port_id, timeout=3)
-                    if isinstance(result, self.dataplane.PollFailure):
-                        self.fail("Expected packet was not received on port %d. Total received: %d.\n%s" % (dst_port_id, cnt, result.format()))
-                    recv_pkt = scapy.Ether(result.packet)
-                    cnt += 1
+                # skip this check for backend ASIC, rely on queue counters
+                # TTL changes on multi ASIC platforms
+                if test_dst_port_name is None:
```

Comment:
instead of bypassing this check for multi asic, we can skip checking testing the ttl ?

Reply:
It's fine with me, @neethajohn Fine to remove TTL check? The exp_ip_id alone is enough to guarantee it's the intended packet.